OWNERS_ALIASES

```diff
@@ -130,6 +130,12 @@ aliases:
     - johnbelamaric
     - klueska
     - pohly
+  wg-etcd-operator-leads:
+    - ahrtr
+    - hakman
+    - jberkus
+    - jmhbnz
+    - justinsb
```

Comment:
If it is applicable I'd like also to be in this list as a representative of our community.

Reply:
Thanks for volunteering for the working group!  This WG isn't a standing body to represent all etcd operators, though.  It's a specific one-time effort to create an official Kubernetes-supported operator.  As part of that, we'll be evaluating existing operators to determine if any of them can become the core of this new operator.  That means that having the author of one such operator making that evaluation, as a WG chair, isn't a good idea.

You are very welcome to participate in the working group once it gets going, including meetings, issues, commentary, and roadmap.